src/ui/language_region_settings_page.ui

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <requires lib="gtk" version="4.0"/>
+  <template class="LanguageRegionSettingsPage" parent="GtkBox">
+    <style><class name="info-box"/></style>
+    <property name="orientation">vertical</property>
+    <property name="vexpand">true</property>
+    <property name="hexpand">true</property>
+    <property name="margin-start">20</property>
+    <property name="margin-end">10</property>
+    <property name="margin-top">10</property>
+    <property name="margin-bottom">10</property>
+    <!--<property name="spacing">10</property>-->
+    <child>
+      <object class="GtkLabel">
+      <property name="label">Language &amp; Region</property>
+      <style><class name="title"/></style>
+      <property name="margin-end">10</property>
+      <property name="margin-top">10</property>
+      <property name="margin-bottom">10</property>
+      <property name="halign">start</property>
+      </object>
+    </child>
+    <child>
+      <object class="GtkBox">
+      <style><class name="info-box"/></style>
+      <property name="orientation">vertical</property>
+      <property name="vexpand">false</property>
+      <property name="hexpand">true</property>
+      <property name="margin-top">10</property>
+      <property name="margin-bottom">10</property>
+      <property name="spacing">10</property>
+        <child>
+          <object class="GtkLabel">
+          <style><class name="normal-text"/></style>
+          <property name="label">Language</property>
+          <property name="halign">start</property>
+          </object>
+        </child>
+        <child>
+          <object class="GtkDropDown" id="language_switch">
+            <style><class name="dropdown-button"/></style>
+            <!--<signal name="notify::selected" handler="on_resolution_changed" swapped="true"/>-->
```

Comment:
I think, it is not needed, switching will be performed on clicking "Apply" button

Reply:
Yeah, copy-paste leftover, will remove.